receiver/azureeventhubreceiver/factory.go

```diff
@@ -21,7 +21,7 @@ const (
 )
 
 var (
-	errUnexpectedConfigurationType = errors.New("Failed to cast configuration to Azure Event Hub Config")
+	errUnexpectedConfigurationType = errors.New("failed to cast configuration to azure event hub config")
```

Comment:
We can do this in a separate PR, ok to leave it in

Reply:
I think my editor may have done this. So I'm just going to leave it for now.